fields/types/s3file/S3FileType.js

```diff
@@ -97,23 +97,25 @@ s3file.prototype.addToSchema = function() {
 
 	var paths = this.paths = {
 		// fields
-		filename:		this._path.append('.filename'),
-		path:			this._path.append('.path'),
-		size:			this._path.append('.size'),
-		filetype:		this._path.append('.filetype'),
-		url:			this._path.append('.url'),
+		filename:   this._path.append('.filename'),
+		path:     this._path.append('.path'),
+		size:     this._path.append('.size'),
+		filetype:   this._path.append('.filetype'),
+		url:      this._path.append('.url'),
+		headers:      this._path.append('.headers'),
 		// virtuals
-		exists:			this._path.append('.exists'),
-		upload:			this._path.append('_upload'),
-		action:			this._path.append('_action')
+		exists:     this._path.append('.exists'),
+		upload:     this._path.append('_upload'),
+		action:     this._path.append('_action')
 	};
 
 	var schemaPaths = this._path.addTo({}, {
-		filename:		String,
-		path:			String,
-		size:			Number,
-		filetype:		String,
-		url:			String
+		filename:   String,
+		path:     String,
+		size:     Number,
+		filetype:   String,
+		url:      String,
+		headers:      Object
```

Comment:
This change isn't necessary as far as I can tell. They headers never come from user-submitted data so they won't change, and we don't need to keep them. (?)